fork-visualizer/src/app/graph/page.js

```diff
@@ -0,0 +1,127 @@
+"use client";
+
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import { useState, useRef } from "react";
+import { ForceGraph2D, ForceGraph3D } from "react-force-graph";
+
+const MAX_CHAIN_COUNT = 256;
+
+// use polkadot main
+// comment out for local
+// const wsProvider = new WsProvider("wss://rpc.polkadot.io");
+
+const wsProvider = new WsProvider("ws://localhost:9944");
+
+function OtherGraph({ data }) {
+    const fgRef = useRef();
+
+    return (
+        <div className="flex-grow">
+            <ForceGraph2D
+                graphData={data}
+                nodeLabel="id"
+                nodeAutoColorBy="group"
+                linkDirectionalArrowLength={3.5}
+                linkDirectionalArrowRelPos={1}
+                ref={fgRef}
+                enableNodeDrag={true}
+                cooldownTicks={100}
+                onEngineStop={() => fgRef.current.zoomToFit(500)}
+            />
+        </div>
+    );
+}
+
+export default function Home() {
+    const [latestBlock, setLatestBlock] = useState();
+    const [data, setData] = useState({ nodes: [], links: [] });
+    const [running, setRunning] = useState(false);
+
+    async function main() {
+        console.log("Starting...")
+
+        // for polkadot main
+        const api = await ApiPromise.create({ provider: wsProvider });
+
+        // for local
+        // const api = await ApiPromise.create();
+
+        // We only display a couple, then unsubscribe
+        let count = 0;
+        setRunning(true);
+        setData({ nodes: [], links: [] });
+
+        // Subscribe to the new headers on-chain. The callback is fired when new headers
+        // are found, the call itself returns a promise with a subscription that can be
+        // used to unsubscribe from the newHead subscription
+        const unsubscribe = await api.rpc.chain.subscribeNewHeads((header) => {
+            setLatestBlock(header.number.toHuman());
```

Comment:
Is `toHuman` the right method here?
Check out https://polkadot.js.org/docs/api/start/types.basics#everything-is-a-type